src/modules/callouts/pages/CreateCallout.vue

```diff
@@ -138,7 +140,22 @@ const steps: Steps = reactive([
   },
 ]);
 
-const submitForm = () => true;
+const makeCalloutData = (creationData: Steps): CreateCalloutData => ({
+  // slug: creationData.find(e => e.name === "URL and sharing")?.data.slug,
+  slug: '',
+  title: '',
+  intro: '',
+  formSchema: { components: [] },
+  excerpt: '',
+  access: 'member',
+  allowUpdate: true,
+  allowMultiple: true,
+  image: 'string',
+  starts: new Date(),
+  expires: new Date(),
+});
+
+const submitForm = () => createCallout(makeCalloutData(steps));
```

Comment:
@wpf500 this is the boilerplate bit I mentioned shamefully. Could also live somewhere else?